### PR TITLE
Fix travis-CI build error

### PR DESCRIPTION
**Pull request by Unode**

### CODE REVIEW

File: Makefile [hunk 1]
```diff
@@ -184,4 +184,7 @@ ngless-${VERSION}.tar.gz: ngless
 	tar -zcvf $(distdir).tar.gz $(distdir)
 	rm -rf $(distdir)
 
-.PHONY: all build clean check tests distclean dist static fast fastcheck modules
+deploy: static
+	ngless_deploy $(shell stack path --dist-dir)/build/ngless/ngless
```
Comment (luispedro) [inline Makefile]: Did you mean to include this commit?

Comment (Unode) [inline Makefile]: Not intentional but also harmless. If you aren't ok with it I'll remove it from the pull request.

Comment (luispedro) [inline Makefile]: Yes, or I can cherry pick the other commit.

Comment (Unode) [inline Makefile]: Fixed

**Pull request merged**